### ISSUE TITLE
Painting Tool

### ISSUE THREAD

**Issue by akash435** [prathimacode-hub/Awesome_Python_Scripts#593]
---
Title: Painting Tool
About: It's a GUI script
Name: Akash Ramanand Rajak
Label: Feature Request
Assignee: ''

---

Define You:

- [x] LGM-SOC'21 Participant 
- [ ] DevIncept Participant
- [ ] Contributor

In this script, will create a painting application using tkinter GUI,

In this will create option like user can select pen, can select any colour for that pen, can increase/decrease the width of line draw through pen, and an eraser to erase the things drawn.

Also will provide a drawing area where user can draw any thing.

Basically will try to make a simple painting tool.

**Comment by akash435** [comment]
@prathimacode-hub @Debashish-hub , please assign this to me under LGM SOC, will add this painting tool under GUIScripts.

**Comment by prathimacode-hub** [comment]
Yes, you can proceed. @akash435
